
```r
# INIT --------------------------------------------------------------------

source("scripts/R-header.R")
set.seed(16341)

```

## Load meta-data
## 1KG phase 3 meta
meta <- readRDS(file = file.path("data/RDS", paste("G1K_meta",".RDS", sep="")))
## prep admixture plotting
options(repr.plot.width=24, repr.plot.height=10)
cols <- pals::tableau20(20)
popCols <- c("#1F77B4", "#FFBFD4", "#82CBFF", "#2CA02C", "#930000", "#000000")

names(popCols) <- c('African','American','East Asian','European','South Asian','Middle East')
ref_pop <- c("European", "East Asian", "American", "South Asian", "African")                                           


# PREPARE PCA AND ADMIXTURE -----------------------------------------------

### extract population SNPs from 1KG and sample VCF
## get 1K Genome vcf and put them back together 
# bcftools concat --output-type z \
# ALL.chr1.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr2.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr3.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr4.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr5.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr6.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr7.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr8.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr9.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr10.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr11.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr12.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr13.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr14.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr15.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr16.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr17.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr18.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr19.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr20.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr21.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# ALL.chr22.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz \
# --output FULL.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz

## also merge the single-sample vcf files
# bcftools merge --file-list merge_samples.txt --output-type z --threads 6 > samples_hg38.vcf.gz 
## remember to index 
# tabix <filename>
## make bed-fiel from Inkens' population snps and extract all the variants
# zgrep -v '^#' WGS_FERNANDES_BUSBY_EUR_AFR_ASIA_FINAL_filtered_0.00.sites > popSNP.bed

## Extract from samples
# bcftools view --regions-file popSNP.bed --output-type z --output samples_hg38_popSNP.vcf.gz samples_hg38.vcf.gz
## aaaand repeat for the 1K Genomes file
# bcftools view --regions-file popSNP.bed --output-type z 1KG_hg38_popSNP.vcf.gz FULL.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz
## if needed bcftools annotate --rename-chrs chrom_mapping.txt input.vcf -o output.vcf
# bcftools annotate --threads 6 --rename-chrs rename.txt --output-type z FULL.shapeit2_integrated_snvindels_v2a_27022019.GRCh38.phased.vcf.gz -o output.vcf.gz

# RUN PCA AND PROJECTION IN PLINK -----------------------------------------

## After joining 1KG and UAE samples we convert to plink and fix the *.fam file to have 6 columns like this:
## Order of sampleIDs matters here, after conversion the original *.fam should have sampleIDs in the first
## two columns. That makes some calculations annoying because they request additional flags --> change.
# sampleID 1 0 0 0 1

## Then we take the first two columns and add a third column with the grouping, i.e., UAE & G1K, and create a 'within_cluster.txt'
## This is used to tell the PCA which Principal components to calculate first.
# sampleID1 1 UAE
# sampleID2 1 G1K

## Then use plink to run PCA and projection (if input is a *.ped this is only --file)
# plink --threads 60 --bfile UAE_G1K_samples --within within_cluster.txt --pca 40 --pca-cluster-names UAE --make-rel --out UAE_G1K_samples_PCA
## This will generate 'UAE_G1K_samples_PCA.eigenval' and 'UAE_G1K_samples_PCA.eigenvec' that can now be processed in R


# RUN SUPERVISED ADMIXTURE ------------------------------------------------

## G1K_meta.RDS contains cleaned up grouping, i.e., source population assignment
## supervised admixture requires a *.pop file that has the same sample order as
## the plink file and contains a single column without header. Every known 
## source population has a label and every unknown sample gets a dash '-'.

## Easiest to use the fam file
fam <- data.table::fread(file.path("data/RAW/plink/Admixture", "plink.fam"))
colnames(fam)[V1] <- "sampleID"
## merge in meta for G1K
out.pop <- fam %>%
  dplyr::left_join(meta, by = "sampleID") %>%
  dplyr::mutate(AdmixPopulation = ifelse(is.na(supPopNameClean),"-", supPopNameClean))

## and save it
data.table::fwrite(list(out.pop$AdmixPopulation), 
                   file.path("data/RAW/plink/Admixture", "plink.pop"),
                   row.names = FALSE, col.names = FALSE)

## and just use:
# admixture --supervised plink.ped 5


# ADMIXTURE IMPORT --------------------------------------------------------

# prep. a vector with names of reference populations
ref_pop <- c("European", "EastAsian", "American", "SouthAsian", "African")

# import admixture output
admix <- data.table::fread(file.path("data/RAW/plink/Admixture", "plink.5.Q"))
fam <- data.table::fread(file.path("data/RAW/plink/Admixture", "plink.fam"))
pop <- data.table::fread(file.path("data/RAW/plink/Admixture", "plink.pop"),  
                         header = F)
# build data.frame
df.admix <- data.frame("sampleID"=fam$V1, "popvec"=pop$V1, admix)
colnames(df.admix) <- c("sampleID","popvec",ref_pop)
rownames(df.admix) <- df.admix$sampleID

# find max. admixture ratio for every sample
max_vec <- apply(df.admix[,c("European", "EastAsian", "American", "SouthAsian", 
                             "African")], 1, which.max)

df.admix <- df.admix %>%
  dplyr::mutate(main_pop = ref_pop[max_vec])

# save it
saveRDS(df.admix, file = 
          file.path("data", "RDS", "Admixture_Q5_meta.RDS"), compress = TRUE)


# PCA IMPORT AND PLOTTING -------------------------------------------------

## load admix info
df.admix <- readRDS(file = file.path("data", "RDS", "Admixture_Q5_meta.RDS"))

## load PCA results and generate plotting data.frames
df.vec <- data.table::fread( file.path("data/RAW/plink/UAE_PCA", 
                                       "UAE_G1K_samples_PCA.eigenvec")) %>%
  dplyr::select(-V2)
df.val <- data.table::fread( file.path("data/RAW/plink/UAE_PCA", 
                                       "UAE_G1K_samples_PCA.eigenval")) 

## generate annotations for plot
metric.df <- getPlinkPCAMetrics(df.val, df.vec)

## make plot df - and also merge in metadata / admix information
plot.df <- df.vec[,1:41] %>% data.frame(stringsAsFactors = FALSE) %>%
  dplyr::rename_at(seq_len(dim(df.vec[,2:41])[2])+1,
                   ~paste("PC",seq_len(dim(df.vec[,2:41])[2]), sep = "")) %>%
  dplyr::rename_at(1, ~"sampleID") %>%
  dplyr::left_join(., df.admix, by = "sampleID") 

## save-it
saveRDS(list(plot.df, metric.df), file = 
          file.path("data", "RDS", "UAE_G1K_PCA.RDS"), compress = TRUE)


## generate plot panel - fill/col can be replaced with grouping variables 
pca12 <- panelPCA(plot.df, metric.df, c("PC1","PC2"), fill.var = "PC1", 
                  col.var = "PC1")
pca23 <- panelPCA(plot.df, metric.df, c("PC2","PC3"), fill.var = "PC2", 
                  col.var = "PC3")
pca34 <- panelPCA(plot.df, metric.df, c("PC3","PC4"), fill.var = "PC3", 
                  col.var = "PC4")

## make a nice panel with letter grouping
pca.plt <- ggarrange(pca12, pca23, pca34, ncol = 3, labels = c("A)", "B)","C)"), 
                     font.label = list(size = 24, color = "black", 
                                       face = "bold", family = NULL))

## save to file
ggsave(file.path("data/output", "PCA_panel_plot.png"), plot = pca.plt, 
       width = 30, height = 11, dpi = "retina")
ggsave(file.path("data/output", "PCA_panel_plot.pdf"), plot = pca.plt, 
       width = 30, height = 11)
```

